spec/acceptance/nodesets/default.yml

```diff
@@ -1,17 +1,31 @@
 ---
 HOSTS:
-  windows2012-64-1:
```

Comment:
modulesync is going to change this back to windows2012 because this module is now marked as cross-platform.  getting around it might be hairy...

Reply:
Thanks Eric, have reverted this.